src/test/java/io/aiven/kafka/connect/gcs/GcsSinkTaskTest.java

```diff
@@ -395,14 +431,42 @@ private SinkRecord createNullRecord(final String topic,
 
     private Collection<String> readRawLinesFromBlob(
         final String blobName,
-        final boolean compressed) {
-        return testBucketAccessor.readLines(blobName, compressed);
+        final String compression) {
+        return testBucketAccessor.readLines(blobName, compression);
     }
 
     private Collection<List<String>> readSplittedAndDecodedLinesFromBlob(
         final String blobName,
-        final boolean compressed,
+        final String compression,
         final int... fieldsToDecode) {
-        return testBucketAccessor.readAndDecodeLines(blobName, compressed, fieldsToDecode);
+        return testBucketAccessor.readAndDecodeLines(blobName, compression, fieldsToDecode);
+    }
+
+    private Map<String, Collection<List<String>>> buildBlobNameValuesMap(final String compression) {
+        final CompressionType compressionType = CompressionType.forName(compression);
+        final String extension = compressionType.extension();
+        final Map<String, Collection<List<String>>> blobNameValuesMap = new HashMap<>();
+        blobNameValuesMap.put("topic0-0-10" + extension, toCollectionOfLists("value0", "value5"));
+        blobNameValuesMap.put("topic0-1-20" + extension, toCollectionOfLists("value1", "value6"));
+        blobNameValuesMap.put("topic1-0-30" + extension, toCollectionOfLists("value2", "value7"));
+        blobNameValuesMap.put("topic1-1-40" + extension, toCollectionOfLists("value3", "value8"));
+        blobNameValuesMap.put("topic0-2-50" + extension, toCollectionOfLists("value4", "value9"));
+        return blobNameValuesMap;
+    }
+
+    /*
+     * example
+     * <pre>
+     * Collection<List<String>> expected = toCollectionOfLists("value0", "value5");
+     * Collection<List<String>> actual = readLinesFromBlob(blob, compression);
+     * assertIterableEquals(expected, actual);
+     * </pre>
```

Comment:
Sorry, when I asked for example, I meant not a usage example, but an example of input-output pair :) Like 
```
Input: "value0", "value5"
Output: Collection[List["value0"], List["value5"]]
```
Could you please add?

Reply:
Done